### DISCUSSION TITLE
SASL ERROR

### DISCUSSION THREAD

**Discussion by corbkris** [go-pg/pg#1924] [General]
I recently upgraded go-pg because I was getting this error "pg: SASL: got \"SCRAM-SHA-256-PLUS\", wanted \"SCRAM-SHA-256\""

https://github.com/go-pg/pg/issues/1483

but now that I updated I get this error “pg: SASL: got ‘S’, wanted ‘\x00’”

could you please provide some insight as to what I’m doing wrong?

**Comment by vmihailenco** [comment]
This is a bug in go-pg. Please try v10.10.5.

Thanks for the report.
